react-time-track/src/components/input-info-project.js

```diff
@@ -2,10 +2,12 @@
 import React from "react";
 import { jsx } from "@emotion/core";
 import { navigate } from "@reach/router";
+import { useAlert } from "react-alert";
 
 import { Button } from "./ui";
 
 function InputInfoProject({ nextFn }) {
+  const alert = useAlert();
   const today = new Date()
```

Comment:
Will look into this as I have to work with dates too. Still we need to learn to set this requirements at the start of the project.

Reply:
Well, working with dates have a little difficulty when you use different formats or do operations, I only saw a specific part of the application which have a little complexity, but the rest is the basic, I know it can become stressful, but you should learn the basic because in many projects the basic is the only necessary, although some libraries help you a lot, but in some cases it doesn't work if your colleagues don't know about that.